package.json

```diff
@@ -1,65 +1,61 @@
 {
-  "name": "eoscrwebappboilerplate",
-  "version": "1.0",
-  "private": true,
-  "dependencies": {
-    "@material-ui/core": "^4.9.12",
-    "@material-ui/icons": "^4.9.1",
-    "@material-ui/lab": "^4.0.0-alpha.51",
-    "@reach/router": "^1.3.3",
-    "clsx": "^1.1.0",
-    "i18next": "^19.4.4",
-    "i18next-browser-languagedetector": "^4.1.1",
-    "prop-types": "^15.7.2",
-    "react": "^16.13.1",
-    "react-dom": "^16.13.1",
-    "react-i18next": "^11.4.0",
-    "react-router": "^5.1.2",
-    "react-router-dom": "^5.1.2",
-    "react-scripts": "^3.4.1",
-    "ual-anchor": "^0.3.2",
-    "ual-ledger": "^0.2.0",
-    "ual-lynx": "^0.3.0",
-    "ual-meetone": "^0.1.1",
-    "ual-reactjs-renderer": "^0.2.0",
-    "ual-scatter": "^0.2.0",
-    "ual-token-pocket": "^0.2.0"
-  },
-  "scripts": {
-    "start": "react-app-rewired start",
-    "build": "react-app-rewired build",
-    "eject": "react-scripts eject",
-    "format": "precise-commits --check-only",
-    "lint": "eslint --ext .jsx,.js src",
-    "test": "react-app-rewired test"
-  },
-  "browserslist": [
-    ">0.2%",
-    "not dead",
-    "not ie <= 11",
-    "not op_mini all"
-  ],
-  "husky": {
-    "hooks": {
-      "pre-commit": "yarn format && yarn lint && CI=true yarn test"
-    }
-  },
-  "devDependencies": {
-    "@babel/plugin-proposal-optional-chaining": "^7.9.0",
-    "customize-cra": "^0.9.1",
-    "eslint": "^6.8.0",
-    "eslint-config-prettier": "^6.11.0",
-    "eslint-config-standard": "^14.1.1",
-    "eslint-config-standard-react": "^9.2.0",
-    "eslint-plugin-import": "^2.20.2",
-    "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^3.1.3",
-    "eslint-plugin-promise": "^4.2.1",
-    "eslint-plugin-react": "^7.19.0",
-    "eslint-plugin-standard": "^4.0.1",
-    "husky": "^4.2.5",
-    "precise-commits": "^1.0.2",
-    "prettier": "^2.0.5",
-    "react-app-rewired": "^2.1.6"
-  }
+	"name": "eoscrwebappboilerplate",
```

Comment:
To keep the same indentation across the project we are using prettier, please install https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode and intent this file with it

Reply:
I think the wrong prettier version. Thanks!